tests/wpt/metadata/dom/events/ProgressEvent.html.ini

```diff
@@ -3,3 +3,4 @@
   bug: https://github.com/servo/servo/issues/10740
   [document.createEvent() should not work with ProgressEvent.]
     expected: FAIL
```

Comment:
Shouldn't this have got removed? Or since it has a `bug:` section, it doesn't automatically get removed?